src/lib/component/fetchAutocompleteFromWs.js

```diff
@@ -1,30 +1,19 @@
-export default function fetchAutocompleteFromWs(
-  term,
-  done,
-  autocompletionWs,
-  localData = []
-) {
-  if (!autocompletionWs) {
-    done(localData)
+export default function fetchAutocompleteFromWs(term, done, autocompletionWs) {
+  if (autocompletionWs) {
+    const url = new URL(autocompletionWs, location)
+    url.searchParams.append('term', term)
+
+    fetch(url.href)
+      .then((response) => {
+        if (response.ok) {
+          return response.json()
+        }
+      })
+      .then((data) => {
+        done(data)
+      })
```

Comment:
```suggestion
      .then(done)
```

引数をそのまま受け取る関数なら、こんな感じで名前だけ指定すれば動きます。
たぶん

Reply:
ローカルで動作確認できました。修正しています。

[Simplify fetchAutocompleteFromWs to pass fetch response directly to done](https://github.com/pubannotation/textae/pull/201/commits/5d8d400d7ff610f290270a934de8a339c8990a44)